.github/workflows/ci.yml

```
@@ -15,7 +15,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [django22-drf312, django30-drf312, django31-drf312, django32-drf312, django32-drflatest, quality, docs]
+        toxenv: [quality, docs, django32-drf312, django32-drflatest, django40-drflatest, django40-drf312]
```

tox environments should be in ascending order, `312`, `latest`
```suggestion
        toxenv: [quality, docs, django32-drf312, django32-drflatest,  django40-drf312, django40-drflatest]
```